defaults/src/main/java/com/yahoo/vespa/defaults/Defaults.java

```diff
@@ -27,15 +27,20 @@
     private final String vespaHost;
     private final int vespaWebServicePort;
     private final int vespaPortBase;
+    private final int vespaPortConfigServerRpc;
+    private final int vespaPortConfigServerHttp;
+    private final int vespaPortConfigProxyRpc;
 
     private Defaults() {
         vespaHome = findVespaHome();
         vespaUser = findVespaUser();
         vespaHost = findVespaHostname();
-        vespaWebServicePort = findVespaWebServicePort();
-        vespaPortBase = 19000; // TODO
+        vespaWebServicePort = findWebServicePort(8080);
+        vespaPortBase = findVespaPortBase(19000);
+        vespaPortConfigServerRpc = findConfigServerPort(vespaPortBase + 70);
+        vespaPortConfigServerHttp = vespaPortConfigServerRpc + 1;
+        vespaPortConfigProxyRpc = findConfigProxyPort(vespaPortBase + 91);
```

Comment:
I think this should be 90

Reply:
right, no idea how i ended up with 91 here...